java/daikon/AnnotateNullable.java

```diff
@@ -134,7 +134,8 @@ public static void main(String[] args) throws IOException {
       if (ppt.is_class()) {
         @SuppressWarnings(
             "nullness") // map: retrieve class name from class Ppt name, with string manipulation
-        @NonNull List<PptTopLevel> static_methods = class_map.get(ppt.name().replace(":::CLASS", ""));
+        @NonNull List<PptTopLevel> static_methods =
```

Comment:
This PR now contains many unrelated formatting changes. Can you undo these?

Reply:
sure.